app/question.js

```diff
@@ -1,19 +1,34 @@
-define(['jquery', 'app/messages', 'app/config', 'app/string'], function ($, messages, config, string) {
-  var currentItemIndex = 0;
-  var inTutorial = config.constant("TUTORIAL_MODE");
+define(['jquery', 'app/messages', 'app/config', 'app/string', 'app/slimstampen'], function ($, messages, config, string, slimstampen) {
   var items;
+  var currentItemIndex = 0;
   var totalLength;
 
+  var itemsAnsweredCorrectly = 0;
+  var answerWasCorrect;
+
+  var tutorialLength;
+  var inTutorial = config.constant("TUTORIAL_MODE");
+
+  var startTime = new Date();
+  var firstKeyPress = 0;
+  var responseList = [];
+
   // Calculate the percentage of 'part out of total'
   function percentage(part, total) {
    return Math.round(part / total * 100);
```

Comment:
You have a percentage function somewhere, might be time to move it to a math module, or something like it.